File: restaurantes/src/main/java/restaurantes/modelo/EventoNuevaValoracion.java
package restaurantes.modelo;

public class EventoNuevaValoracion {
    private String idOpinion;
    private Valoracion nuevaValoracion;
    private int numeroValoraciones;
    private double calificacionMedia;
    
	public String getIdOpinion() {
		return idOpinion;
	}
	public void setIdOpinion(String idOpinion) {
		this.idOpinion = idOpinion;
	}
	public Valoracion getNuevaValoracion() {
		return nuevaValoracion;
	}
	public void setNuevaValoracion(Valoracion nuevaValoracion) {
		this.nuevaValoracion = nuevaValoracion;
	}
	public int getNumeroValoraciones() {
		return numeroValoraciones;
	}
	public void setNumeroValoraciones(int numeroValoraciones) {
		this.numeroValoraciones = numeroValoraciones;
	}
	public double getCalificacionMedia() {
		return calificacionMedia;
	}
	public void setCalificacionMedia(double calificacionMedia) {
		this.calificacionMedia = calificacionMedia;
	}
}

File: opiniones-rest/Models/EventoNuevaValoracion.cs
using opiniones_rest.Modelo;

namespace opiniones_rest.Eventos
{
    public class EventoNuevaValoracion
    {
        public string IdOpinion { get; set; }
        public Valoracion NuevaValoracion { get; set; }
        public int NumeroValoraciones { get; set; }
        public double CalificacionMedia { get; set; }
    }
}